pom.xml

```diff
@@ -35,6 +39,13 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
             <version>2.3</version>
+            <exclusions>
+                <!-- To avoid ClassNotFoundException during InjectedTest -->
+                <exclusion>
+                    <groupId>org.apache.httpcomponents</groupId>
+                    <artifactId>httpclient</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Yes, I need it because `git` plugin includes a old version of `httpclient` and ATH needs **its** version.